src/AutoTable.jsx

```diff
@@ -21,6 +21,20 @@ ErrorRow.propTypes = {
   columnCount: PropTypes.number.isRequired
 };
 
+const NoResultsRow = ({ columnCount }) => (
+  <TableRow>
+    <TableCell colSpan={columnCount}>
+      <Typography variant="caption" align="center">
+        No results found.
```

Comment:
The ability to pass a custom no results found message could be nice here.

Reply:
yessir handled by the renderProps 😃